### DISCUSSION TITLE
Backbone model frozen with fusion_transformer/fusion_mlp

### DISCUSSION THREAD

**Discussion by nimasteryang** [autogluon/autogluon#3475] [Q&A]
When I try to leverage the fusion_transformer/fusion_mlp with different backbone models (i.e. with difference model.hf_text.checkpoint_name), is there any way I can freeze the backbone model during the training so that Autogluon will use those backbone models as some kind of feature extractors and only tune the transformer/mlp layer without altering backbone models' parameters?🤔

**Comment by AldoCP** [comment]
@nimasteryang Did you find a solution for this? 

Or, relatedly, does anybody know how to freeze some layers from `model.hf_text`? Namely, is there anything like the `frozen_layers` parameter used for `mmdet_image` below, but for `hf_text`? This would be a nice feature... 😉 

https://auto.gluon.ai/stable/tutorials/multimodal/advanced_topics/customization.html#model-mmdet-image-frozen-layers

**Reply by nimasteryang** [reply]
not yet...
